packages/block-editor/src/components/spacing-sizes-control/all-input-control.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Internal dependencies
+ */
+import SpacingInputControl from './spacing-input-control';
+import { ALL_SIDES, getAllRawValue } from './utils';
+
+export default function AllInputControl( {
+	onChange,
+	values,
+	sides,
+	spacingSizes,
+} ) {
+	const allValue = getAllRawValue( values );
+	// const hasValues = isValuesDefined( values );
+	// const isMixed = hasValues && isValuesMixed( values, sides );
```

Comment:
Todo: need to plumb this up so user gets some indication the current sides have mixed valued